tests/utils.go

```diff
@@ -478,7 +478,7 @@ func WaitForAllPodsReady(timeout time.Duration, listOptions metav1.ListOptions)
 		virtClient, err := kubecli.GetKubevirtClient()
 		PanicOnError(err)
 
-		podsList, err := virtClient.CoreV1().Pods(k8sv1.NamespaceAll).List(listOptions)
+		podsList, err := virtClient.CoreV1().Pods(KubeVirtInstallNamespace).List(listOptions)
```

Comment:
i suggest to print out any not running pods anyway, but only fail on specific pods:
1. pods under the kubevirt NS as was done in this PR
2. other NMs we care deeply about e.g etcd, mco, ca related etc (but im not sure how easy it is to do for different platforms)